packages/backend/src/services/organisme/ProtocoleSanitaire.js

```diff
@@ -206,17 +206,20 @@ module.exports.createOrUpdate = async (organismeId, parametre) => {
     await client.query(query.removeRAM, [protocoleSanitaireId]);
 
     const responsableAdministrationMedicament =
-      parametre.responsableAdministrationMedicament;
-    if (responsableAdministrationMedicament.length > 0) {
+      parametre.responsableAdministrationMedicament ?? null;
+    if (
+      responsableAdministrationMedicament &&
+      responsableAdministrationMedicament.length > 0
+    ) {
       await client.query(
         query.associateRAM(responsableAdministrationMedicament.length),
         [protocoleSanitaireId, ...responsableAdministrationMedicament],
       );
     }
 
     await client.query(query.removeCE, [protocoleSanitaireId]);
-    const constitutionEquipe = parametre.constitutionEquipe;
-    if (constitutionEquipe.length > 0) {
+    const constitutionEquipe = parametre.constitutionEquipe ?? null;
+    if (constitutionEquipe && constitutionEquipe.length > 0) {
```

Comment:
```suggestion
    if (constitutionEquipe?.length > 0) {
```

si tu veux un one liner